### PR TITLE
Update Init for Facebook

### PR DESCRIPTION
**Pull request by nitrag**
Updating for swift 3

### CODE REVIEW

**Comment by nitrag** [comment]
Signed CLA.

File: en/ios/users.mdown [hunk 1]
```diff
@@ -485,10 +485,19 @@ There's also two code changes you'll need to make. First, add the following to y
 import FBSDKCoreKit
 import ParseFacebookUtilsV4
 
-// AppDelegate.swift
-func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
-  Parse.setApplicationId("parseAppId", clientKey:"parseClientKey")
-  PFFacebookUtils.initializeFacebookWithApplicationLaunchOptions(launchOptions)
+// AppDelegate.swift (Swift 3)
+func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -&gt; Bool {
```
Comment (hramos) [inline en/ios/users.mdown]: Is \&gt\; necessary here? It should render fine within the code block, and it looks like you didn't need to use this encoding later in the same file.

**Review by hramos — CHANGES_REQUESTED** [review]

**Comment by TomWFox** [comment]
I'm sorry that this has been over looked. Would you be able to update this code for swift 4.2 (+ the latest version of the Parse iOS SDK) and then I will take another look?

Also would you be able to remove the vertical parameter alignment as it doesn't appear well on the docs:
screenshot 2019-03-08 at 11 58 12

**Comment by TomWFox** [comment]
I'm closing this now as I have started a new pull request (#594) that brings in these changes (those of which are still needed).

**Pull request closed without merging**